scraping.py

```diff
@@ -17,16 +17,23 @@ def scrape_files_num(rows):
 
     for row in rows:
         _row: str = row.html
-        file_num   = extract_file_num(_row)
+        file_num   = strip_file_num(_row)
```

Comment:
> __[Pylint]__ Exactly one space required before assignment ([view](https://sider.review/gh/repos/291573177/pulls/5#issue-4477965))
>
> <details>
>         file_num   = strip_file_num(_row)
>                    ^
>
> | Rule | Severity | Message-id | Module | Obj |
> | ---- | -------- | ---------- | ------ | --- |
> | `bad-whitespace` | `convention` | C0326 | scraping | - |
>
> You can [close this issue](https://sider.review/gh/repos/291573177/pulls/5#issue-4477965) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

doesn't need fix it because the spaces are For readability